packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -2009,8 +2010,12 @@ function AccountHack(props) {
   );
 }
 
-export default function Account(props) {
+export default function Account() {
   const syncEnabled = useFeatureFlag('syncAccount');
+  let params = useParams();
+  let location = useLocation();
+  let activeLocation = useActiveLocation();
```

Comment:
nit: `const`

```suggestion
  const params = useParams();
  const location = useLocation();
  const activeLocation = useActiveLocation();
```

Reply:
We should pick one and add a lint rule. I think James pretty consistently used let and IMO we should either ban const or enforce using it whenever possible.

Reply:
Makes sense. My vote would be for `const` :) 

`let` is too loose. Allows to re-assign values.

Reply:
In the words of Ryan Florence: "I'm too old to type an extra letter." 😆 

Just kidding. I tend to match the style of the existing function, but I reflexively reach for `const`, too. (Although the actual meaning of "re-assign" seems to trick a lot of folks.)

_Let_'s choose a lint rule separately. I agree the dichotomy is just more mental overhead, slight though it may be.